cmd/lightchain/simulate.go

```diff
@@ -94,7 +94,7 @@ func simulateTransferTx(nodeCfg node.Config) *types.Transaction {
 		panic(err)
 	}
 
-	tx, err := wallety.Transfer(client, auth, common.HexToAddress(standaloneNonExistingAddr), simulateTxAmount.String(), txclient.NewTransferTxConfig())
+	tx, err := wallety.Transfer(client, auth, common.HexToAddress(standaloneNonExistingAddr), simulateTxAmount.String(), txclient.NewTransferTxConfig(database.MinGasPrice))
```

Comment:
[comment] I had to modify the declaration of MinGasPrice to not run in recursive imports

Reply:
Redeclaring appears to me to be less scalable than passing same argument over an over again. Anyway this should be move later on into a lightchain config object, do you mind to postpone that decision till then?